core/storage/base_model/gae_models_test.py

```diff
@@ -236,6 +236,39 @@ def test_get_unversioned_instance_id(self):
         model1.put()
         self.assertEqual(model1.get_unversioned_instance_id(), 'model_id')
 
+    def test_export_data_trivial(self):
+        user_data = (base_models
+                     .BaseSnapshotMetadataModel
+                     .export_data('trivial_user'))
+        expected_data = {}
+        self.assertEqual(user_data, expected_data)
+
+    def test_export_data_nontrivial(self):
+        version_model = TestVersionedModel(id='version_model')
+        model1 = version_model.SNAPSHOT_METADATA_CLASS.create(
+            'model_id-1', 'committer_id', 'create', None, None)
+        model1.put()
+        model2 = version_model.SNAPSHOT_METADATA_CLASS.create(
+            'model_id-2', 'committer_id', 'create', 'Hi this is a commit.',
+            None)
```

Comment:
Perhaps use something that's not None for commit_cmds here, to check that it actually gets exported properly.

Reply:
Done